### PR TITLE
Allow parsing atomics

### LINKED ISSUE #1: Feature Request: LL(1) parser to create an AST for atomic expressions
### Problem Description
As a software developer I expect "slang" to provide expressions with similar syntax and semantics as in other programming languages. A grammar as well as the corresponding functions of a recursive descent parser are already available. Apart from my sample implementation for literals, however, all functions currently throw a runtime exception with the message "not yet implemented".

### Your Team Mission
All methods relevant to this task are marked as `TODO` and the associated grammar rules are available as JavaDoc. Find out which methods are responsible for the atomic expressions and adjust them to create a correct abstract syntax tree (AST). You can take a test-driven approach. There is a separate test class for each team. Your team is considered successful if all unit tests of your test class succeed. The remaining tests should continue to fail. This is intentional, as it is the job of your fellow students to fulfill these tests.

### PR DESCRIPTION
**Pull request by jens1o**
fixes #1

### CODE REVIEW

File: src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java [hunk 1]
```diff
@@ -363,26 +364,59 @@ public AstNode exponent2( AstNode previous ){
...
 	
 	@Override
 	public AstNode atomicExpression1( ){
-		// TODO Auto-generated method stub
-		return super.atomicExpression1( );
+		if (this.LEXER.lookahead().TYPE != LPAREN) {
+			throw this.parsingException(LPAREN);
+		}
+
+		this.LEXER.advance();
```
Comment (marco-haupt) [inline src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java:387]:
```suggestion
		match( Token.Type.LPAREN );
```

File: src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java [hunk 2]
```diff
@@ -363,26 +364,59 @@ public AstNode exponent2( AstNode previous ){
...
+		AstNode node = this.conditionalExpression();
+		if (this.LEXER.lookahead().TYPE != RPAREN) {
+			throw this.parsingException(RPAREN);
+		}
+		this.LEXER.advance();
```
Comment (marco-haupt) [inline src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java:392]:
```suggestion
		match( Token.Type.RPAREN );
```

**Review by marco-haupt — CHANGES_REQUESTED** [review]
Well done! :)

File: src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java [hunk 3]
```diff
@@ -363,26 +364,52 @@ public AstNode exponent2( AstNode previous ){
...
-		return super.atomicExpression2( );
+		if (this.LEXER.lookahead().TYPE != IDENTIFIER) {
+			throw this.parsingException(IDENTIFIER);
+		}
+
+		AstVariable variable = new AstVariable(this.LEXER.lookahead().LEXEME);
+
+		this.LEXER.advance();
+
+		return variable;
```
Comment (marco-haupt) [inline src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java:400]: Why are there so many blank lines?

**Review by marco-haupt — APPROVED** [review]
Let's overlook the blank lines this time...
All tests of the corresponding test class are successful. Thank you for your contribution!

File: src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java (hunk 3 above)
Comment (jens1o) [reply src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java:400]: well.... good point :D

Feel free to remove them in a follow-up. :)

**Pull request merged**